ansible/roles/commcarehq/tasks/main.yml

```diff
@@ -55,6 +55,7 @@
       pip:
         requirements: "{{ code_home }}/requirements/{{ item }}"
         virtualenv: "{{ virtualenv_source }}"
+        chdir: "{{ code_home }}"
```

Comment:
Need to have this because requirements now have a requirement from a submodule

https://github.com/dimagi/commcare-hq/blob/bb238246bf98a5fe6f3702bee038404b03d2b073/requirements/requirements.txt#L101